src/middlewares/auth.js

```diff
@@ -16,51 +17,73 @@ function unauthorized(res) {
 }
 
 export default (scopes) => {
+  // For now we only allow 'admin' scope.
+  const validScopes = ['admin', 'client', 'user'].filter(
+    scope => scopes.includes(scope)
+  );
+
+  if (!validScopes.length) {
+    throw new Error(`No valid scope found in "${scopes}"`);
+  }
+
   return (req, res, next) => {
-    const authHeader = req.headers['authorization'];
-    if (!authHeader) {
+    const authHeader = req.headers.authorization;
+    const authQuery = req.query.authorizationToken;
```

Comment:
To be able to use a simple browser request, we can pass the JWT token in this query parameter. I haven't looked if there is anything kinda standardized for this yet.